python/build-deps.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -e
+
+BUILD_JOBS=$1
+
+export CMAKE_GENERATOR=Ninja
```

Comment:
Is this crucial, or just convenient?

Reply:
convenience. it simply means i don't have to pass it to every cmake invocation. using ninja instead of make is a significant speedup.